sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -1143,8 +1143,7 @@
     },
     "/messages/serviceBound/feedback": {
       "get": {
-        "summary": "This method is used to retrieve feedback of a cloud-to-device message.",
-        "description": "This method is used to retrieve feedback of a cloud-to-device message See https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-messaging for more information. This capability is only available in the standard tier IoT Hub. For more information, see [Choose the right IoT Hub tier](https://aka.ms/scaleyouriotsolution). For IoT Hub VNET related features(https://docs.microsoft.com/en-us/azure/iot-hub/virtual-network-support) please use API version '2020-03-13'.These features are currently in general availability in the East US, West US 2, and Southcentral US regions only. We are actively working to expand the availability of these features to all regions by end of month May. For rest of the APIs please continue using API version '2019-10-01'",
+        "description": "This method is used to retrieve feedback of a cloud-to-device message See https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-messaging for more information. This capability is only available in the standard tier IoT Hub. For more information, see [Choose the right IoT Hub tier](https://aka.ms/scaleyouriotsolution).",
```

Comment:
Do these comments support markdown syntax?

Reply:
I don't know that.

Reply:
This may be the swagger friendly way to introduce relevant links: https://idratherbewriting.com/learnapidoc/pubapis_openapi_step8_externaldocs_object.html

Reply:
I'd be curious how this looks once the code is generated

Reply:
Interesting find Tim. Do we want to go this way with externalDocs ?